src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,78 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Nested;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CalculatorTest {
+
+    private final Calculator calculator = new Calculator();
+
+    @Nested
+    @DisplayName("덧셈 테스트")
+    class addTest {
+
+        @Test
+        @DisplayName("정수의 합")
+        void add_Test() {
+            final int a = 3;
+            final int b = 5;
+            final int actual = a+b;
+
+            assertEquals(actual, calculator.add(a,b));
+        }
+    }
+
+    @Nested
+    @DisplayName("뺄셈 데스트")
+    class subtractTest {
+
+        @Test
+        @DisplayName("정수의 차")
```

Comment:
각 테스트 메서드가 어떤 상황을 테스트하는 지 구체적으로 설명하면 좋을 것 같아요!
```suggestion
        @DisplayName("각 정수의 뺸 값을 반환한다.")
```

또한, 뺄셈의  '각 정수의 뺄셈 시 음수가 나올 경우에는 어떻게 처리할 것인지'와 같은 다양한 상황에 대해서도 고려해보시면 더 좋을 것 같습니다!

Reply:
명사형이 아닌 문장형으로 테스트 상황을 구체적으로 서술하는 것이 중요하다는 점 이해했습니다.
또한, 제안해주신 예외 상황을 뺄셈 뿐만 아니라 다른 연산 테스트에도 적용하여 다양한 상황을 고려하도록 하겠습니다. 감사합니다!